common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -31,63 +35,79 @@ function Log-Warning($message) {
     }
 }
 
-Register-PackageSource `
-    -Name $NIGHTLY_FEED_NAME `
-    -Location $NIGHTLY_FEED_URL `
-    -ProviderName Nuget `
-    -ErrorAction SilentlyContinue `
+function GetAllPackages {
+    $packages = Find-Package -Source $PACKAGE_FEED_URL -AllVersion -AllowPrereleaseVersions
+    if ($Daily) {
+        return $packages | Where-Object { $_.Version.Contains("alpha") }
```

Comment:
Should this also check for "dev", or are none left that we smoke test?

Reply:
Ah good point. The sorting algorithm will never pick dev anyway, but it would be good to exclude here just to be defensive.

Reply:
Misread this comment. We don't want any dev packages, as they should all be alpha now. I did add an exclusion for dev when `Daily == false`. @weshaggard to double check on this.